Taskfile.yml

```diff
@@ -293,25 +295,35 @@ tasks:
       COMMIT:
         sh: cat internal/util/version/gen/commit.txt
 
-  nfpm_deb:
+  packages:
```

Comment:
This one used to call `packager`, right? I guess it's renamed on purpose. I see it's called from `.github/workflows/packages.yml`, so it seems to be fine.

Reply:
Yeah, and we don't document / promote it as stable yet